### ISSUE TITLE
Adding support for pipewire

### ISSUE THREAD

**Issue by sthibaul** [espeak-ng/pcaudiolib#12]
Hello,
pipewire is very likely to replace pulseaudio on the desktop, so it'd be useful for pcaudiolib to support it.
Samuel

**Comment by sthibaul** [comment]
> As pipewire is intended to be drop-in replacement of pulseaudio (i.e. supporting wrapper for pulseaudio clients)

Yes, indeed. But the more I was looking at improving the pulseaudio support in speech-dispatcher, the more I was finding the pulseaudio interface limitating, and preventing proper support such as audio progress tracking and such. From looking at who works on pipewire, I'd tend to think that pipewire provides a better interface for e.g. audio progress.

**Comment by valdisvi** [comment]
As far as I learned about pipewire, it definitely is better than undocumented mess called pulseaudio.
